librocksdb_sys/build.rs

```diff
@@ -131,7 +131,6 @@ fn build_rocksdb() -> Build {
     let mut cfg = Config::new("rocksdb");
     if cfg!(feature = "encryption") {
         cfg.register_dep("OPENSSL").define("WITH_OPENSSL", "ON");
-        println!("cargo:rustc-link-lib=static=crypto");
```

Comment:
maybe provide a extra feature to control whether to static link openssl?

Reply:
It's not necessary, users can statically link openssl by specifying `openssl/vendored` feature on their own crates. See https://github.com/tikv/tikv/pull/15944 .